stock_mts_mto_rule/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{'name': 'Stock MTS+MTO Rule',
+ 'version': '12.0.1.0.0',
+ 'author': 'Hibou Corp.,Akretion,Odoo Community Association (OCA)',
+ 'website': 'http://www.akretion.com',
```

Comment:
Could you follow OCA conventions for manifest (website, missing entires) ?

https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py